src/public/view/Reset/Reset.tsx

```diff
@@ -70,13 +74,19 @@ export function Reset() {
         <Heading level={1} alignSelf='self-start'>
           Reset Password
         </Heading>
-        <ResetForm
-          token={resetToken}
-          inProgress={reset.isPending}
-          onSubmit={reset.mutate}
-        />
+        {info.isPending ? (
```

Comment:
This is wrong. You should show form when info is successful and when submit is successful i.e. `reset.isSuccessful`, then show `ResetPasswordSuccessful` component.

Looks like, the conditions are not tested.